app/models/user.rb

```diff
@@ -9,12 +9,33 @@ class User < ApplicationRecord
   validate :validate_image_type
 
   has_one_attached :profile_picture
-  has_many :posts
+  has_many :posts # , dependent:   :destroy ??
```

Comment:
Yes, should be dependent destroy. We don't have the ability to delete a user, so technically it's not necessary